data-migration/migrate-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,273 @@
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    nohup tiup tidb-lightning -config tidb-lightning.toml > nohup.out &
+    ```
+
+3. After TiDB Lightning completes the import, it exits automatically. If the import is successful, the last line of `tidb-lightning.log` prints `tidb lightning exit`.
+
+If the import fails, refer to [TiDB Lightning FAQ](/tidb-lightning/tidb-lightning-faq.md) for troubleshooting.
+
+## Step 3: Replicate incremental data to TiDB
+
+### Add the data source
+
+1. Create a `source1.yaml` file as follows:
+
+    {{< copyable "" >}}
+
+    ```yaml
+    # Configuration.
+    source-id: "mysql-01" # Must be unique.
+
+    # Configures whether DM-worker uses the global transaction identifier (GTID) to pull binlogs. To enable this mode, the upstream MySQL must also enable GTID. If the upstream MySQL has automatic source-replica switching, then GTID mode is required.
+    enable-gtid: false
+
+    from:
+      host: "${host}"           # e.g.: 172.16.10.81
+      user: "root"
+      password: "${password}"   # Supports but not recommended to use plaintext password. It is recommended to use `dmctl encrypt` to encrypt the plaintext password before using it.
+      port: 3306
+    ```
+
+2. Load the data source configuration to the DM cluster using `tiup dmctl` by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    tiup dmctl --master-addr ${advertise-addr} operate-source create source1.yaml
+    ```
+
+    The options used in the command above are described as follows:
+
+    |Parameter              |Description    |
+    |-                      |-              |
+    |`--master-addr`        |The {advertise-addr} of any DM-master in the cluster that `dmctl` is connecting to, e.g.: 172.16.10.71:8261|
+    |`operate-source create`|Load the data source to the DM cluster.|
+
+### Add a replication task
+
+1. Edit the `task.yaml` file. Configure the incremental replication mode and the starting point of each data source:
+
+    {{< copyable "shell-regular" >}}
+
+    ```yaml
+    name: task-test                      # Task name. Must be globally unique.
+    task-mode: incremental               # Task mode. The "incremental" mode nly performs incremental data replication.
+
+    # Configures the target TiDB database.
+    target-database:                     # The target database instance.
+      host: "${host}"                    # e.g.: 127.0.0.1
+      port: 4000
+      user: "root"
+      password: "${password}"            # It is recommended to use `dmctl encrypt` to encrypt the plaintext password before using it.
+
+    # Use block and allow lists to specify the tables to be replicated.
+    block-allow-list:                    # The collection of filtering rules that matches the tables in the source database instance. If the DM version is v2.0.0-beta.2 or earlier, use black-white-list instead.
```

Comment:
I think we can remove the `If the DM version is v2.0.0-beta.2 or earlier, use black-white-list instead` here. I think there is no `v2.0.0-beta.2` dm in tiup.